docs/infrastructure-memcached.md

```diff
@@ -22,18 +22,10 @@ View the [memcached exporter](prometheus-memcached-exporter.md) instructions to
 kubectl --namespace openstack get horizontalpodautoscaler.autoscaling memcached -w
 ```
 
-### Create shared os-memcached secret
-
-``` shell
-kubectl --namespace openstack \
-        create secret generic os-memcached \
-        --type Opaque \
-        --from-literal=memcache_secret_key="$(< /dev/urandom tr -dc _A-Za-z0-9 | head -c${1:-32};echo;)"
-```
-
 !!! Note
 
-    This is a shared secret that is distributed to all services that require it. Rotating this value means updating all services.
+    This os-memcached shared secret is distributed to all services that require it.
```

Comment:
note needs to be updated if you intend on removing the example shell command to create the secret.